### DISCUSSION TITLE
请问怎么从geosite.dat里排除某个域名

### DISCUSSION THREAD

**Discussion by PHCSJC** [IrineSistiana/mosdns#349] [Q&A] [answered]
匹配本地域名这一块，里面有个域名我想排除掉，因为这个域名实际解析出的IP是在国外，请问怎么排除这个域名呢？
  - tag: query_is_local_domain
    type: query_matcher
    args:
      domain:
        - 'ext:./geosite.dat:cn'

**Comment by kiuber** [comment]
+1

**Comment by kiuber** [comment, marked as answer]
@PHCSJC 我用 if_and 和 '!tag' 可以做到排除
